### ISSUE TITLE
Accessability Violation : The ARIA attributes "aria-label" are not valid for the element <div> with implicit ARIA role "generic"（ibm-table-toolbar）

### ISSUE THREAD

**Issue by GSSCDylan** [carbon-design-system/carbon-components-angular#2551]
## Detailed description

> Describe in detail the issue you're having.

The ARIA attributes "aria-label" are not valid for the element

with implicit ARIA role "generic"
> Is this a feature request (new component, new icon), a bug, or a general issue?

bug
> Is this issue related to a specific component?

ibm-table-toolbar
> What did you expect to happen? What happened instead? What would you like to see changed?

Need to remove aria-label in div tag of  ibm-table-toolbar
> What browser are you working in?

Firefox
> What version of the Carbon Design System are you using?

"carbon-components": "^10.58.3",
"carbon-components-angular": "^4.71.8",

https://angular.carbondesignsystem.com/?path=/story/components-table--with-toolbar

## Additional information

* Screenshots or code
<img width="1752" alt="action tool bar" src="https://user-images.githubusercontent.com/73825731/237058731-1424f7ae-581f-425f-a119-327f9d5ee663.png">
<img width="1732" alt="action tool bar2" src="https://user-images.githubusercontent.com/73825731/237059423-dad8ec19-4128-4151-b2f1-f7c5772e2a95.png">
